package.json

```diff
@@ -14,10 +14,9 @@
     "axios": "^0.18.1",
     "core-js": "^3.6.4",
     "css-loader": "^0.28.11",
-    "launchdarkly-js-client-sdk": "^2.17.0",
     "lodash": "^4.17.15",
     "regenerator-runtime": "^0.13.3",
-    "sbc-common-components": "2.0.41",
+    "sbc-common-components": "^2.1.8",
```

Comment:
Don't use ^ ... we don't want to automatically upgrade to the latest version. We want to control when we upgrade as there is extra regression testing to do.